tool/travis.sh

```diff
@@ -24,30 +24,28 @@ tuneup check --ignore-infos
 pub global activate webdev
 webdev build
 
-if [ "$USE_FLUTTER_SDK" = true ] ; then
-    # Get Flutter.
-    curl https://storage.googleapis.com/flutter_infra/releases/stable/macos/flutter_macos_v1.0.0-stable.zip -o ../flutter.zip
-    cd ..
-    unzip -qq flutter.zip
-    export PATH=`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin:$PATH
-    flutter config --no-analytics
-    flutter doctor
-
-    # Should be using dart from ../flutter/bin/cache/dart-sdk/bin/dart
-    echo "which dart: " `which dart`
-
-    # Return to the devtools directory
-    cd devtools
-
-    # Run the tests that require the Flutter SDK.
-    pub run test -t "useFlutterSdk"
-
-    # Chrome test passes locally but fails on Travis. See example failure:
-    # https://travis-ci.org/flutter/devtools/jobs/472755560.
-    # TODO: investigate if we have a need to run tests requiring the Flutter SDK on Chrome.
-    # pub run test -t "useFlutterSdk" -pchrome-no-sandbox
-else
-    # Run the tests that do not require the Flutter SDK.
-    pub run test -x "useFlutterSdk"
-    pub run test -x "useFlutterSdk" -pchrome-no-sandbox
-fi
+# Run tests that do not require the Flutter SDK.
+pub run test -x "useFlutterSdk"
```

Comment:
nit: I would expect these double quotes are not needed.

Reply:
removed.